src/ivc/cyclefold/sfc/input/mod.rs

```diff
@@ -408,7 +576,121 @@ impl<const ARITY: usize, F: PrimeField, RO: ROTrait<F>> AbsorbInRO<F, RO> for In
 
 impl<const ARITY: usize, F: PrimeField> Input<ARITY, F> {
     pub(super) fn get_without_witness(&self) -> Self {
```

Comment:
Just to double confirm, this is used for initial input at step zero?

Reply:
Yes, in pp.new